packages/node/src/parsers.ts

```diff
@@ -2,9 +2,19 @@ import { SentryEvent, SentryException, StackFrame } from '@sentry/types';
 import { readFileAsync } from '@sentry/utils/fs';
 import { basename, dirname } from '@sentry/utils/path';
 import { snipLine } from '@sentry/utils/string';
+import * as Limiter from 'async-limiter';
+import { LRUMap } from 'lru_map';
 import * as stacktrace from 'stack-trace';
+import { NodeOptions } from './backend';
 
-const LINES_OF_CONTEXT: number = 7;
+// tslint:disable-next-line:no-unsafe-any
+const fsLimiter = new Limiter({ concurrency: 25 });
```

Comment:
Do we need a limiter if we have a cache though? Not sure

Reply:
Right, I removed it.